main.c

```diff
@@ -47,27 +52,36 @@ int main(void) {
     CANRXPPS = 0x11;
 
     // set up CAN module
-    can_timing_t can_setup;
-    can_generate_timing_params(_XTAL_FREQ, &can_setup);
+    can_timing_t can_setup = {
+        .brp      = 47,
+        .sjw      =  3,
+        .btlmode  =  1,
+        .sam      =  0,
+        .seg1ph   =  4,
+        .prseg    =  0,
+        .seg2ph   =  4
+    };
```

Comment:
this is Ok for now, but if you could please add 48Mhz as a supported frequency to can_genreate_timing params, and open a PR for that, that would be appreciated as that is the correct way to do this.